Header header
float32 longitude
float32 latitude
float32 yaw
float32 velocity
float32 yawRate
float32 longAcceleration